static/app/components/searchQueryBuilder/index.spec.tsx

```diff
@@ -326,7 +326,6 @@ describe('SearchQueryBuilder', function () {
         'placeholder',
         '-1d'
       );
-      await userEvent.click(screen.getByRole('combobox', {name: 'Edit filter value'}));
```

Comment:
These lines now caused the test to fail (as they should) because clicking the combobox again should toggle it closed. Because we were missing the `state.isOpen` check these tests were passing.